{
  "messages": [
    {
      "error_id": "",
      "references": [
        {
          "column": 9,
          "file": "/home/yo30qaqy/data-race-detection-benchmark-suite/results-20250730-140308/SPMDIR-MUST/conflict/007-MPI-conflict-get-get-local-yes.c",
          "line": 39
        },
        {
          "column": 9,
          "file": "/home/yo30qaqy/data-race-detection-benchmark-suite/results-20250730-140308/SPMDIR-MUST/conflict/007-MPI-conflict-get-get-local-yes.c",
          "line": 41
        }
      ],
      "text": "Found get operation/call at results-20250730-140308/SPMDIR-MUST/conflict/007-MPI-conflict-get-get-local-yes.c:39:9 which is in conflict with get operation/call at results-20250730-140308/SPMDIR-MUST/conflict/007-MPI-conflict-get-get-local-yes.c:41:9 before release",
      "type": "Local Data Race"
    }
  ]
}
